src/Compilers/Server/VBCSCompiler/NamedPipeClientConnectionHost.cs

```diff
@@ -94,7 +94,26 @@ public void EndListening()
             Debug.Assert(_queue is object);
             Debug.Assert(_listenTasks is object);
 
-            _cancellationTokenSource.Cancel();
+            try
+            {
+                // Even though the Tasks created to run the compilation servers can never throw, 
+                // the CancellationToken from this source ends up getting passeed throughout the 
```

Comment:
>passeed [](start = 74, length = 7)

Typo

Reply:
Thanks. Will pick that up in the next change here. Merging now so  I can get rid of the flaky test.